ludwig/hyperopt/execution.py

```diff
@@ -896,10 +896,10 @@ def _register(name, trainable):
             result_grid = tuner.fit()
             # Get ExperimentAnalysis object from ResultGrid
             analysis = result_grid._experiment_analysis
-        except Exception as e:
+        except Exception:
             # Explicitly raise a RuntimeError if an error is encountered during a Ray trial.
             # NOTE: Cascading the exception with "raise _ from e" still results in hanging.
-            raise RuntimeError(f"Encountered Ray Tune error: {e}")
+            raise RuntimeError(f"Encountered Ray Tune error: {traceback.format_exc()}")
```

Comment:
You can change this to:

```
raise RuntimeError(...) from e
```

That way you get the whole traceback without needing to turn it into a stirng.